server/controllers/short-format/courtDetailsController.test.ts

```diff
@@ -12,6 +12,11 @@ jest.mock('../../services/reportService', () => {
           process.nextTick(() => resolve({}))
         })
       },
+      updateReport: () => {
+        return new Promise(resolve => {
```

Comment:
This boilerplate has been added to a lot of tests, would it be possible to extract it to a helper class or similar?

Reply:
Yeah, we could move this to an external mock. It's a bit of a PITA but I'll look into it.

Reply:
I've created external mocks for the services that were manually mocked in the tests now to remove/reduce duplication.